ch1/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8" />
```

Comment:
```suggestion
    <meta charset="utf-8">
```

HTML5에서는 스스로 닫히는 태그가 있을 때 `/`를 생략하는 편입니다.